### ISSUE TITLE
Author mode: a parameter node looked like an unknown on the canvas (missing "=#")

### ISSUE THREAD

**Issue by jonwetzel** [Dragoon-Lab/topomath#311]
We haven't figured out the reproduction steps for this yet.  Happened to two different students, both using chrome on macbooks.

**Comment by jonwetzel** [comment]
Today's class this happened with a parameter node called "DTom" for the following three users.  They created DTom in author mode on the following problems.  Section will be non class models.   Note difference in casing.

User: aortizst
Folder: classwork
Model: Lunch break

User: davaz
Folder: Classwork
Model: Lunch Break

User: Davis
Folder: Classwork
Model: lunch break

**Comment by jonwetzel** [comment]
@dharanisakary  I have figured out how to reproduce this bug!

1. Click "Add Quantity"
1. Enter a description
1. Enter a variable name
1. Click Parameter
1. Enter a value
1. Click Done
1. Open the node again
1. Click Unknown
1. Click Parameter (value is now empty--I guess this is the correct behavior)
1. Enter a value, then click done without hitting tab first (don't make the value box turn blue)

On clicking done, the value field should get verified and turn blue before the node editor closes, but it does not.

**Comment by dharanisakary** [comment]
Forgot to update the replication steps in the card. Had mentioned in my daily update.
For author mode parameter node looking like unknown - 
I followed the steps mentioned in the ticket, but dint find the issue, however, checking logs - I found that when we change the name of parameter, the node looks like unknown but saved as parameter.
Steps to Reproduce - 
1. Create a quantity node.
2. Set description
3. Set Variable Name, say 'a'
4. Set the variable Type as 'Paramater'
5. Enter value say 20
6. Close the node
7. Open the node and change Variable name to say 'b'
8. Node is displayed as b on the canvas , but not b= 20
